src/pyodide/BUILD.bazel

```diff
@@ -1,7 +1,42 @@
 load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
-load("@bazel_skylib//rules:copy_directory.bzl", "copy_directory")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
-load("@workerd//:build/wd_js_bundle.bzl", "wd_js_bundle")
+load("@capnp-cpp//src/capnp:cc_capnp_library.bzl", "cc_capnp_library")
+load("//:build/capnp_embed.bzl", "capnp_embed")
+load("//:build/wd_js_bundle.bzl", "wd_js_bundle")
+
+copy_file(
+    name = "pyodide_packages_archive",
+    src = "@pyodide_packages//:pyodide_packages_unzipped_0.1.tar",
+    out = "generated/pyodide_packages_unzipped_0.1.tar",
+)
+
+capnp_embed(
+    name = "pyodide_packages_archive_embed",
+    src = "generated/pyodide_packages_unzipped_0.1.tar",
+    deps = ["pyodide_packages_archive"],
+)
+
+copy_file(
+    name = "packages_capnp_file",
+    src = "pyodide_packages.capnp",
+    out = "generated/pyodide_packages.capnp",
+)
+
+capnp_embed(
+    name = "packages_capnp_file_embed",
+    src = "generated/pyodide_packages.capnp",
+    deps = ["packages_capnp_file"],
+)
+
+cc_capnp_library(
+    name = "pyodide_packages_capnp",
+    srcs = ["generated/pyodide_packages.capnp"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":packages_capnp_file_embed",
+        ":pyodide_packages_archive_embed",
+    ],
+)
```

Comment:
Are these changes necessary? From what I can tell you should be able to call `getPyodideEmbeddedPackages` in `PackagesTarReader`. Would avoid the need for defining a new capnp library manually.

Reply:
I didn't really like the vibes of sticking it in a list of packages with a `moduleName` when it isn't intended to be imported as a module, and then we have to iterate the BUNDLE to fish it back out. It seems like a hack. Why not just put it in its own separate capnp variable?